microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/FaultToleranceCommand.java

```diff
@@ -57,6 +58,7 @@
 public class FaultToleranceCommand extends HystrixCommand<Object> {
     private static final Logger LOGGER = Logger.getLogger(FaultToleranceCommand.class.getName());
 
+    static final int MAX_THREAD_WAITING_PERIOD = 2000;      // milliseconds
```

Comment:
Should this max be configurable?

Reply:
Yes, there's actually a few others like this. I'll create another issue to address this later.